icepyx/core/exceptions.py

```diff
@@ -3,16 +3,12 @@ class DeprecationError(Exception):
     Class raised for use of functionality that is no longer supported by icepyx.
     """
 
-    pass
-
 
 class QueryError(Exception):
     """
     Base class for Query object exceptions
     """
 
-    pass
-
```

Comment:
why isn't this needed (isn't the class empty otherwise)?

Reply:
When a class has a docstring, it doesn't need a placeholder like `pass` or `...`.

Reply:
Fascinating. I had no idea. Thanks for teaching me!